src/components/CookieConsentBanner.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import ReactGA from 'react-ga'
+import Cookies from 'js-cookie'
+
+import { c_PRIMARY_TEXT, c_PRIMARY_BACKGROUND_ALTERNATIVE } from '../theme'
+
+const Button = ({ onClick, className, children, open }) => (
+  <button type="button" onClick={onClick} className={className}>
+    {children}
+  </button>
+)
+
+const ACCEPTANCE_COOKIE = 'neontribe-cookies-accept'
+const NEONTRIBE_GA = 'UA-000000-01'
+
+const startGA = () => {
+  ReactGA.initialize(NEONTRIBE_GA)
+}
+
+const stopGA = () => {
+  ReactGA.ga('remove')
+  Cookies.remove('_ga')
+  Cookies.remove('_gid')
+  Cookies.remove('_gat')
+}
+
+const CookieConsentBanner = () => {
+  const [isBannerOpen, setIsBannerOpen] = React.useState(false)
+  const [hasTrackedView, setHasTrackedView] = React.useState(false)
+  const [acceptanceCookie, setAcceptanceCookie] = React.useState(
+    Cookies.get(ACCEPTANCE_COOKIE)
+  )
+
+  React.useEffect(() => {
+    if (acceptanceCookie === 'true') {
+      startGA()
+    }
+
+    if (acceptanceCookie !== 'true') {
+      stopGA()
+    }
+  }, [acceptanceCookie])
+
+  React.useEffect(() => {
+    if (window.ga && !hasTrackedView) {
+      ReactGA.pageview(window.location.pathname + window.location.search)
+      setHasTrackedView(true)
+    }
+  }, [hasTrackedView, acceptanceCookie])
+
+  const updateAcceptanceCookie = () => {
+    setIsBannerOpen(false)
+    setAcceptanceCookie(Cookies.get(ACCEPTANCE_COOKIE))
+  }
+
+  const acceptCookies = () => {
+    Cookies.set(ACCEPTANCE_COOKIE, true, { expires: 28 * 3 })
+    updateAcceptanceCookie()
+  }
+
+  const declineCookies = () => {
+    Cookies.set(ACCEPTANCE_COOKIE, false)
+    updateAcceptanceCookie()
+  }
+
+  const acceptanceCookieExists = typeof acceptanceCookie !== 'undefined'
+
+  const displayBanner = !acceptanceCookieExists || isBannerOpen
+
+  return (
+    <div>
+      <div aria-expanded={displayBanner}>
+        {displayBanner && (
+          <div className="cookie-banner">
+            <h1 id="cookie-content-p1">
+              Can we use cookies to help us improve this site?
+            </h1>
```

Comment:
This shouldn't be an h1 as there will already be an h1 on the page. It doesn't look like the id is in use, so this can be removed.